README.md

```diff
@@ -84,7 +84,24 @@ docker run --rm -it --net cbio-net \
     migrate_db.py -p /cbioportal/portal.properties -s /cbioportal/db-scripts/src/main/resources/migration.sql
 ```
 
-### Step 5 - Run the cBioPortal web server ###
+### Step 5 - Build Session Service containers
```

Comment:
For people who have experience with Docker, the word **build** suggests you're compiling an image based on a Dockerfile, and not running any services yet. The verb for starting services/processes in containers based on existing images is **run**. At a glance, that made me miss these steps when looking for how to start the services.

Reply:
This was a mistake from my side. I've fixed it. Good catch 😃